### PR TITLE
Canberra Adkins form

### PR DESCRIPTION
**Pull request by wasade**
This pull request introduces Canberra-Adkins, which is a normalized variant of the Canberra metric. This metric is more similar, although not identical, to the "Canberra" metric of QIIME1. The slight difference stems from a bug in the QIIME1 implementation in which the number of nonzero elements used in the scaling factor was over estimated. 

Co-authors on this include: @gregcaporaso @antgonza @ElDeveloper @rob-knight and @justin212k following internal discussion. The discrepancy with QIIME1 and QIIME2 was first identified by Dr. Alexey Melnik.

TODO: 

* Is the `ValueError` necessary, or does the `validate=True` in `beta_diversity` assure the values are positive?

### CODE REVIEW

File: q2_diversity/_beta/_method.py [hunk 1]
```diff
@@ -42,7 +43,7 @@ def non_phylogenetic_metrics():
             'correlation', 'hamming', 'jaccard', 'chebyshev', 'canberra',
             'braycurtis', 'mahalanobis', 'yule', 'matching', 'dice',
             'kulsinski', 'rogerstanimoto', 'russellrao', 'sokalmichener',
-            'sokalsneath', 'wminkowski', 'aitchison'}
+            'sokalsneath', 'wminkowski', 'aitchison', 'canberra-adkins'}
```
Comment (ebolyen) [inline q2_diversity/_beta/_method.py]: Super minor, but the other metrics (really just UniFrac) seem to use `_` in their spelling.

Comment (thermokarst) [reply q2_diversity/_beta/_method.py]:
```suggestion
            'sokalsneath', 'wminkowski', 'aitchison', 'canberra_adkins'}
```

Comment (wasade) [reply q2_diversity/_beta/_method.py]: :+1:, applied the suggestion

**Comment by thermokarst** [comment]
note to self - before merging remember to add in the co-author tags for the [aforementioned authors](https://github.com/qiime2/q2-diversity/pull/221#issue-219082485).

File: q2_diversity/_beta/_method.py [hunk 2]
```diff
@@ -124,9 +125,24 @@ def beta(table: biom.Table, metric: str,
     def aitchison(x, y, **kwds):
         return euclidean(clr(x), clr(y))
 
+    def canberra_adkins(x, y, **kwds):
+        # Table 1 of Faith et al 1987 Plant Ecology 69: 57-68
```
[the remark this answers is not in the document]
Comment (wasade) [reply q2_diversity/_beta/_method.py]: Added, but wasn't sure if or where to add the a link to the citation?

**Comment by wasade** [comment]
Ideally it would go into scipy but we decided it would be a lot faster to get it working here than to try and add to scipy. skbio is certainly an intermediate option.

File: q2_diversity/_beta/_method.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (thermokarst) [reply q2_diversity/_beta/_method.py]: You can add a citation in the plugin registration, for example: https://github.com/qiime2/q2-diversity/blob/master/q2_diversity/plugin_setup.py#L126-L130

[the remark this answers is not in the document]
Comment (thermokarst) [reply q2_diversity/_beta/_method.py]: Yeah, but we have been erring on the side of over-citing in this plugin by tacking them onto the method (for now, as a stop-gap measure).

**Pull request merged**